helpdesk-bot/src/test/java/org/symphonyoss/symphony/bots/helpdesk/bot/it/steps/TicketSteps.java

```diff
@@ -111,8 +134,42 @@ public void sendInitialQuestion(String user)
     Thread.sleep(5000L);
   }
 
+  @When("$user sends another initial question to the bot")
```

Comment:
I think the message is duplicated, if you need send another message, please pass with param.

Reply:
ok